### PR TITLE
runelite: 1.6.0 -> 2.0.0

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Motivation for this change

2.0.2 (latest) doesn't provide a `RuneLite.jar`.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @Kmeakin

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/74136)

### CODE REVIEW

**Comment by filalex77** [comment]
@GrahamcOfBorg build runelite

File: pkgs/games/runelite/default.nix [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+  version = "2.0.0";
 
   src = fetchurl {
     url = "https://github.com/runelite/launcher/releases/download/${version}/RuneLite.jar";
-    sha256 = "0q2xx0wrnlg5mrv8nnmnh300r8mqfm8k2p028m7mr09kn18xvkzx";
+    sha256 = "0q0x8g43ggkvp5fgnymgygx59xfhnyyrhpi6ha333gqg97rr0rvw";
   };
 
   icon = fetchurl {
     url = "https://github.com/runelite/runelite/raw/master/runelite-client/src/main/resources/runelite.png";
```
Comment (marsam) [inline pkgs/games/runelite/default.nix]: would you mind updating the url to use the tag url?
```suggestion
     url = "https://github.com/runelite/runelite/raw/${version}/runelite-client/src/main/resources/runelite.png";
```

**Comment by filalex77** [comment]
Thanks @marsam!

**Comment by marsam** [comment]
Thanks you @filalex77

**Pull request merged**